### ISSUE TITLE
Vieb-7.1.0.AppImage FUSE issue

### ISSUE THREAD

**Issue by veikolippand** [Jelmerro/Vieb#355]
Not sure if it's bug or what but Vieb-7.1.0.AppImage is not working on my machine. There is no issues with older versions. Vieb-7.0.0.AppImage is currently in use and no problems.
```

$ ./Vieb-7.1.0.AppImage
Squashfs image uses (null) compression, this version supports only xz, zlib.

Cannot mount AppImage, please check your FUSE setup.
You might still be able to extract the contents of this AppImage
if you run it with the --appimage-extract option.
See https://github.com/AppImage/AppImageKit/wiki/FUSE
for more information
open dir error: No such file or directory

```
Is there any changes in AppImage building or where this FUSE issue game in?
```
106839069 Nov 29 09:31 /opt/Vieb-6.2.0.AppImage
107724238 Jan 24 10:27 /opt/Vieb-7.0.0.AppImage
 97298235 Feb  8 13:49 /opt/Vieb-7.1.0.AppImage
```
There is changes in file size as well, so maybe something is missing or simply broken?

**Comment by Jelmerro** [comment]
Seems to be [an electron builder bug](https://github.com/electron-userland/electron-builder/issues/6624). I'll check if it's possible to use a different version, I had to update to the alpha channel a while ago, but hopefully the fix I was waiting for is in stable by now.

**Comment by Jelmerro** [comment]
Fixed in 7.1.1 :tada:
